diff --git a/multiple_fields_remove_button.module b/multiple_fields_remove_button.module
index d05620f..50df74c 100644
--- a/multiple_fields_remove_button.module
+++ b/multiple_fields_remove_button.module
@@ -119,6 +119,7 @@ function multiple_fields_remove_button_field_widget_single_element_form_alter(&$
     'email',
     'entity_reference',
     'entity_reference_autocomplete',
+    'entity_reference_revisions',
     'float',
     'integer',
     'link',
