diff --git a/src/Form/FormsStepsAlter.php b/src/Form/FormsStepsAlter.php
index c39ca69..874f0db 100644
--- a/src/Form/FormsStepsAlter.php
+++ b/src/Form/FormsStepsAlter.php
@@ -158,7 +158,7 @@ class FormsStepsAlter {
     $jumpStepEvent = new StepChangeEvent($forms_steps, $step, $jumpStep, $form_state);
 
     $event_dispatcher = \Drupal::service('event_dispatcher');
-    $event_dispatcher->dispatch(StepChangeEvent::STEP_CHANGE_EVENT, $jumpStepEvent);
+    $event_dispatcher->dispatch($jumpStepEvent, StepChangeEvent::STEP_CHANGE_EVENT);
   }
 
   /**
